### PR TITLE
Adding a LoadKeychain function that can pull data from an ini

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: go/probe/go.mod [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module github.com/preludeorg/detect-clients/go/probe
 
-go 1.19
+go 1.16
```
Comment (privateducky) [inline go/probe/go.mod:3]: what prompted this?

File: go/probe/pkg/service/service.go [hunk 2]
```diff
@@ -1,37 +1,42 @@
...
 	Register(string)
+	LoadKeychain()
 }
 
 func CreateService() *ProbeService {
 	return &ProbeService{
-		token:         os.Getenv("PRELUDE_TOKEN"),
-		accountId:     os.Getenv("PRELUDE_ACCOUNT_ID"),
-		accountSecret: os.Getenv("PRELUDE_ACCOUNT_SECRET"),
+		HQ:            util.GetEnv("PRELUDE_HQ", "https://detect.prelude.org"),
```
Comment (privateducky) [inline go/probe/pkg/service/service.go:30]: makes sense to shift here

**Review by privateducky — COMMENTED** [review]
all good, as usual, just make sure I see these pre-merge (otherwise I could miss it in the PR snowstorm here).

**Comment by khyberspache** [comment]
Needed it for a demo at 5 with Anduril

**Comment by privateducky** [comment]
> Needed it for a demo at 5 with a customer

Totally, works fine this time. Just keep in mind, I'm staying available to pair on this stuff to to move quicker.

**Pull request merged**